src/processes/registerv2-process.js

```diff
@@ -69,6 +69,7 @@ const registerProcess = async (job, done) => {
     }
 
     await ProcessHelper.followUser(userId, follows);
+    await ProcessHelper.createEmptyChannel(userId, follows);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createEmptyChannel` function is called with `userId` and `follows` as parameters, but there's no error handling in case this function fails. It would be better to wrap this call in a try-catch block to handle any potential errors.
```diff
-    await ProcessHelper.createEmptyChannel(userId, follows);
+    try {
+        await ProcessHelper.createEmptyChannel(userId, follows);
+    } catch (error) {
+        console.error(`Failed to create empty channel for user ${userId}: `, error);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->